qa/suites/fs/functional/tasks/snap-schedule.yaml

```diff
@@ -15,6 +15,7 @@ overrides:
       - is full \(reached quota
       - POOL_FULL
       - POOL_BACKFILLFULL
+      - cluster \[WRN\] evicting unresponsive client
```

Comment:
You did mention in https://tracker.ceph.com/issues/66009#note-35 that the mount is unresponsive in the mgr. What is the reason for the unresponsiveness?

Reply:
That unresponsiveness is due to mgr respawning without formally closing client connections before respawn.

Reply:
oh, and given that we respawn everything after each test, this would show up even more, yes?

Reply:
yes, that's right

Reply:
should I move this item to a more generic `log-ignorelist` ?

Reply:
The mgr respawning is due to enabling/disabling the snap-schedule module, no?

Reply:
yeah, but the mgr across respawns discards cephfs clients ... whereas the mds eventually notices a client without any activity from among the older discarded clients and throws up this warning